templates/config/i18n-tasks.yml

```diff
@@ -116,6 +116,17 @@ search:
 #   # OpenAI
 #   openai_api_key: "sk-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 #   # openai_model: "gpt-3.5-turbo" # see https://platform.openai.com/docs/models
+#   # openai_system_prompt: # may contain `%{from}` and `%{to}`, which will be replaced by source and target locale codes, respectively, using `Kernel.format`
```

Comment:
The settings in this file are formatted sothat uncommenting turns it into a valid option
Please move the comment above the line and add the appropriate multiline string sigil.

Reply:
I just changed it as I think you requested :+1:

I tried to make it consistent with what was done for Deepl configuration, for both mandatory configuration lines, optional (commented) configuration lines and plain old comments (with plain old English).

I used a [folded style](https://yaml.org/spec/1.2.2/#813-folded-style) indicator (`>`) with a  "stripping" [block chomping](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator) indicator (`-`) to explicitly indicate the multiline field and how it will be processed (resulting in a `>-` marker): it seems to me it was the more logical processing method to apply, but I'm happy to change it if you prefer another one.

I hope that's what you expected.

------------------

On a side note, though, I'm pretty sure the previous implementation was still valid YAML when uncommented (despite the end-of-line comment and the lack of explicit multiline indicator :shrug:):

```ruby
YAML.safe_load(<<~YAML)
  openai_system_prompt: # may contain `%{from}` and `%{to}`, which will be replaced by source and target locale codes, respectively (using `Kernel.format`)
    You are a professional translator that translates content from the %{from} locale
    to the %{to} locale in an i18n locale array.
  
    The array has a structured format and contains multiple strings. Your task is to translate
    each of these strings and create a new array with the translated strings.
  
    HTML markups (enclosed in < and > characters) must not be changed under any circumstance.
    Variables (starting with %%{ and ending with }) must not be changed under any circumstance.
  
    Keep in mind the context of all the strings for a more accurate translation.
YAML
# => {"openai_system_prompt"=>
#      "You are a professional translator that translates content from the %{from} locale to the %{to} locale in an i18n locale array.\nThe array has a structured format and contains multiple strings. Your task is to translate each of these strings and create a new array with the translated strings.\nHTML markups (enclosed in < and > characters) must not be changed under any circumstance. Variables (starting with %%{ and ending with }) must not be changed under any circumstance.\nKeep in mind the context of all the strings for a more accurate translation."}
```

YAML is such a pain :trollface:

Reply:
Indeed, YAML is gnarly. Thanks!

Reply:
Thanks to you for all the work on this gem :pray: :purple_heart: